From d88a121aab0036709b2e0149d8c9db204b2da7a0 Mon Sep 17 00:00:00 2001
From: rbuschow <rbuschow@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 25 Aug 2016 12:43:34 +0000
Subject: [PATCH 0786/1076] MANTIS 41953 AEA Conveyor DeviceHandler

add additional methods in aeaDevice.idl for aea conveyor:
RC prepare()
RC reset()

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1076 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/aeaDevice.idl | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/SelfServiceCommon/Massai/idl/Massai/aeaDevice.idl b/SelfServiceCommon/Massai/idl/Massai/aeaDevice.idl
index 962d226a..2e875b35 100644
--- a/SelfServiceCommon/Massai/idl/Massai/aeaDevice.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/aeaDevice.idl
@@ -71,9 +71,15 @@ module aeaDevice
     
     interface servant
     {
+        /** Prepare a device session */
+        RC prepare();
+        
         /** Request a status indication */
         RC requestStatus();
     
+        /** Reset a device session */
+        RC reset();
+        
         /** Sets a listener to this device handler.
          *  @param il - The indication listener
          *  @return a return code as specified in this module */        
-- 
2.41.0.windows.1

